conf/themes/silverstripe/class.twig

```diff
@@ -1,47 +1,38 @@
-{% extends "layout/layout.twig" %}
-{% from "macros.twig" import render_classes, breadcrumbs, namespace_link, class_link, property_link, method_link, hint_link, source_link, method_source_link, deprecated, deprecations, todo, todos, class_category_name %}
-{% block title %}{{ class|raw }} | {{ parent() }}{% endblock %}
-{% block body_class 'class' %}
-{% block page_id 'class:' ~ (class.name|replace({'\\': '_'})) %}
-
-{% block below_menu %}
-    {% if class.namespace %}
-        <div class="namespace-breadcrumbs">
-            <ol class="breadcrumb">
-                <li><span class="label label-default">{{ class_category_name(class.getCategoryId()) }}</span></li>
-                {{ breadcrumbs(class.namespace) -}}
-                <li>{{ class.shortname|raw }}</li>
-            </ol>
-        </div>
-    {% endif %}
-{% endblock %}
+{% extends "default/class.twig" %}
+{% from "macros.twig" import render_classes, property_link, method_link, hint_link, deprecated, deprecations, internals, todos %}
 
+{# This has to be included so that we can add config after constants #}
 {% block page_content %}
 
     <div class="page-header">
         <h1>
-            {{ class.name|split('\\')|last|raw }}
-            {{ deprecated(class) }}
+            {{- class.name|split('\\')|last|raw -}}
+            {{- deprecated(class) }}
+            {% if class.isReadOnly() %}<small><span class="label label-primary">{% trans 'read-only' %}</span></small>{% endif -%}
         </h1>
     </div>
 
+    {% if class.hasSince() %}
+        <i>{{ 'Since:'|trans }} {{ class.getSince() }}</i>
```

Comment:
We should probably support this upstream

Reply:
Or maybe you did sync the files and it's already done 😄

Reply:
Yeah that's straight from the dictum twig file. The only project-specific changes here are clearly marked with comments.

Reply:
Thanks, it's lovely to have such comments